charts/longhorn/100.1.3+up1.2.5/Chart.yaml

```diff
@@ -0,0 +1,41 @@
+annotations:
+  catalog.cattle.io/auto-install: longhorn-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Longhorn
+  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.25.0-0'
+  catalog.cattle.io/namespace: longhorn-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/provides-gvr: longhorn.io/v1beta1
+  catalog.cattle.io/rancher-version: '>= 2.6.0-0 < 2.7.0-0'
+  catalog.cattle.io/release-name: longhorn
+  catalog.cattle.io/type: cluster-tool
+  catalog.cattle.io/ui-component: longhorn
+  catalog.cattle.io/upstream-version: 1.2.5
+apiVersion: v1
+appVersion: v1.2.5
+description: Longhorn is a distributed block storage system for Kubernetes.
+home: https://github.com/longhorn/longhorn
+icon: https://raw.githubusercontent.com/cncf/artwork/master/projects/longhorn/icon/color/longhorn-icon-color.png
+keywords:
+- longhorn
+- storage
+- distributed
+- block
+- device
+- iscsi
+- nfs
+kubeVersion: '>=1.18.0-0'
```

Comment:
Not just 1.2, 1.3 also does not support K8s 1.25. 

We should do this limitation for upstream charts, but they have been released, so we should add this when we have 1.2.6 and 1.3.2 later. 

This is for the Rancher chart, so it's fine because it has been limited by the Rancher chart mechanism.